src/pages/gateway/cicd.md

```diff
@@ -89,9 +89,9 @@ CLIENTID: ${{ secrets.CLIENTID_PROD }}
 CLIENTSECRET: ${{ secrets.CLIENTSECRET_PROD }}
 TECHNICALACCOUNTID: ${{ secrets.TECHNICALACCID_PROD }}
 TECHNICALACCOUNTEMAIL: ${{ secrets.TECHNICALACCEMAIL_PROD }}
-IMSORGID: ${{ secrets.IMSORGID_PROD }}
-ORGID: ${{ secrets.ORGID_PROD }}
-PROJECTID: ${{ secrets.PROJECTID_PROD }}
+IMSORGID: ${{ secrets.IMSORGID }}
+ORGID: ${{ secrets.ORGID }}
+PROJECTID: ${{ secrets.PROJECTID }}
```

Comment:
@nuzil if you change these variables, you will need to change the other references to these variables in the `yml` scripts in this file. also, i believe it is possible that a developer could have multiple orgs and projects that they could designate as production or stage.

Reply:
about 1st point - you are right, forgot about it.
about 2nd - you also right :-) but its confused me on first setup, so can also others, I guess usecase you describing is less then 5% of all usecases, I do not see a reason for now why will I have another org and project for same mesh just as another workspace, as I can create as much as I want to on single one.
But up to you

Reply:
yes, i think you are correct, it is probably an edge case, and would be more clear without the environment specification.

Reply:
@nuzil can you make the changes to the variable names in the scripts in your branch?